RecoTracker/MkFit/plugins/MkFitSeedConverter.cc

```diff
@@ -147,7 +147,7 @@ mkfit::TrackVec MkFitSeedConverter::convertSeeds(const edm::View<TrajectorySeed>
         LogTrace("MkFitSeedConverter") << " adding hit detid " << detId.rawId() << " index " << clusterRef.index()
                                        << " ilay " << ilay;
         ret.back().addHitIdx(clusterRef.index(), ilay, 0);  // per-hit chi2 is not known
-      } else {
+      } else if (!mkFitGeom.isPhase2()) {                   // XXXX why is !phase2 if needed here?
```

Comment:
to be resolved before submission. I think there should at least be an "else" with a throw for the remaining option

Reply:
Removed.